src/ansys/mapdl/core/_commands/database/selecting.py

```diff
@@ -20,1853 +20,2021 @@
+            documentation
+            <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_KSEL.html>`_ for further
+            information.
 
-        VMAX
-            Maximum value of item range. VMAX defaults to VMIN.
-
-        VINC
-            Value increment within range. Used only with integer ranges (such as for area numbers). Defaults
-            to 1. VINC cannot be negative.
-
-        KSWP
-            Specifies whether only areas are to be selected:
-
-                - `kswp = 0`
-                  Select areas only.
-                - `kswp = 1`
-                  Select areas, as well as keypoints, lines, nodes, and elements associated with selected areas.
-
-            Valid only with Type = S.
+        vinc : str
+            The description of the argument is missing in the Python function. Please, refer to the `command
+            documentation
+            <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_KSEL.html>`_ for further
+            information.
 
+        kabs : str
+            The description of the argument is missing in the Python function. Please, refer to the `command
+            documentation
+            <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_KSEL.html>`_ for further
+            information.
 
         Notes
         -----
-        Selects a subset of areas. For example, to select those areas with area
-        numbers 1 through 7, use
 
-        >>> mapdl.asel('S','AREA', '', 1, 7)
+        .. _KSEL_notes:
 
-        The selected subset is then used when the ALL label is entered
-        (or implied) on other commands, such as
+        Selects a subset of keypoints or hard points. For example, to select a new set of keypoints based on
+        keypoint numbers 1 through 7, use :ref:`ksel`,S,KP,,1,7. The selected subset is used when the ALL
+        label is entered (or implied) on other commands, such as :ref:`klist`,ALL. Only data identified by
+        keypoint number are selected. Data are flagged as selected and unselected; no data are actually
+        deleted from the database.
 
-        >>> mapdl.alist('ALL')
+        This command is valid in any processor.
 
-        Only data identified by area number are selected.  Data
-        are flagged as selected and unselected; no data are actually deleted
-        from the database.
+        For selections based on non-integer numbers (coordinates, results, etc.), items that are within the
+        range ``VMIN`` - ``Toler`` and ``VMAX`` + ``Toler`` are selected. The default tolerance ``Toler`` is
+        based on the relative values of ``VMIN`` and ``VMAX`` as follows:
+
+        If ``VMIN`` = ``VMAX``, ``Toler`` = 0.005 x ``VMIN``.
+        If ``VMIN`` = ``VMAX`` = 0.0, ``Toler`` = 1.0E-6.
+        If ``VMAX`` ≠ ``VMIN``, ``Toler`` = 1.0E-8 x ( ``VMAX`` - ``VMIN`` ).
```

Comment:
it might be a good idea to make this bullet points, or similar. Otherwise, sphinx will render it as just one line.